d2/src/main/proto/XdsD2.proto

```diff
@@ -40,5 +40,55 @@ message D2SymlinkNode {
 }
 
 message D2NodeMap {
+  // Deprecated in favor of D2UriMap
+  option deprecated = true;
   map<string, D2Node> nodes = 1;
 }
+
+// D2URI is a proto representation of com.linkedin.d2.balancer.properties.UriProperties. Note that a D2 UriProperties is
```

Comment:
why we have announcement from a single machine for potentially multiple URIs?  is this i001, and i002 multiple instances on a single machine? 
the original name UriProperties is a plural, we would need plural to be consistent with UriProperties?:)

Reply:
This is a legacy feature, I'm not sure if it's actually ever used, but we can't take that chance. As far as the naming goes, it's the properties of 1 "URI" since it's one URI ZK node, but the code itself suggests there can be more than one uri within the URI node. It's terrible naming all around but the naming of this message is consistent with the rest of the code

Reply:
It would be more comforting to know exactly for what purpose this feature existed before getting rid of it.

Reply:
Nvm, just checked many of the URIs using some scripting and none of them seems to be using multiple URIs for one ephemeral node.